test/cypress/support/commands.js

```diff
@@ -34,4 +34,10 @@
 //       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
 //     }
 //   }
-// }
\ No newline at end of file
+// }
+
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
You should target specific exceptions here - we do not want to miss an exception that is avoidable - please see the following for an example - 

```
    Cypress.on('uncaught:exception', (err) => {
      // returning false here prevents Cypress from
      // failing the test
      if (err.message.includes("this.facets.at is not a function"))
        return false
      if (err.message.includes("Cannot read properties of undefined (reading 'left')"))
        return false
      return true
    })
```